hack/local-up-cluster.sh

```diff
@@ -233,7 +233,7 @@ CONTAINER_RUNTIME_ENDPOINT=${CONTAINER_RUNTIME_ENDPOINT:-""}
 IMAGE_SERVICE_ENDPOINT=${IMAGE_SERVICE_ENDPOINT:-""}
 CHAOS_CHANCE=${CHAOS_CHANCE:-0.0}
 CPU_CFS_QUOTA=${CPU_CFS_QUOTA:-true}
-ENABLE_HOSTPATH_PROVISIONER=${ENABLE_HOSTPATH_PROVISIONER:-"false"}
```

Comment:
You should enable it only when you create the default storage class for hostpath, i.e. somewhere in `create_storage_class`. The script deploys real AWS/GCE/OpenStack/... storage class in the clouds and it does not need hostpath provisioner there (unless user enables it explicitly via env. variable).